src/freemindlatex/compilation_service.py

```diff
@@ -0,0 +1,74 @@
+import json
```

Comment:
This is really a service interface definition. We should have a _service, _server_lib, server_main.